source/features/deep-reblame.tsx

```diff
@@ -0,0 +1,126 @@
+import './deep-reblame.css';
+import mem from 'mem';
+import React from 'dom-chef';
+import select from 'select-dom';
+import blurAccessibly from 'blur-accessibly';
+import versionIcon from 'octicon/versions.svg';
+import delegate, {DelegateEvent} from 'delegate-it';
+import * as api from '../libs/api';
+import features from '../libs/features';
+import loadingIcon from '../libs/icon-loading';
+import {getRepoGQL, getReference, looseParseInt} from '../libs/utils';
+
+const getPullRequestBlameCommit = mem(async (commit: string, prNumber: number): Promise<string | false> => {
+	const {repository} = await api.v4(`
+		repository(${getRepoGQL()}) {
+			object(expression: "${commit}") {
+				... on Commit {
+					associatedPullRequests(last: 1) {
+						nodes {
+							number
+							mergeCommit {
+								oid
+							}
+							commits(last: 1) {
+								nodes {
+									commit {
+										oid
+									}
+								}
+							}
+						}
+					}
+				}
+			}
+		}
+	`);
+
+	const associatedPR = repository.object.associatedPullRequests.nodes[0];
+	const mergeCommit = associatedPR.mergeCommit.oid;
```

Comment:
Errors: https://github.com/sindresorhus/refined-github/blame/d1e27cb34f18b9d5d57fdbf853e5b15c3a660d85/source/features/deep-reblame.tsx#L67

```
Uncaught (in promise) TypeError: Cannot read property 'oid' of null
```

Maybe this is all that's needed, test it

```suggestion
	const mergeCommit = associatedPR.mergeCommit?.oid;
```

Reply:
Good catch

Reply:
Needed a full is statement. since associatedPR returns undefined